.travis.yml

```diff
@@ -67,7 +67,7 @@ install:
     ( set -euo pipefail
       git clone https://github.com/NlightNFotis/cbmc.git --branch "external_parser_support_rebased" --single-branch ${CBMC}
       cd ${CBMC}
-      cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release'  '-DCMAKE_CXX_COMPILER=/usr/lib/ccache/g++-7'
+      cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release'  '-DCMAKE_CXX_COMPILER=g++-7'
```

Comment:
Do you know what `g++-7` is on the PATH? I had a sneaky feeling that some of the travis configs put the ccache wrappers on the path before the actual compiler binaries... I could be wrong though. Specifying an absolute path is safer, and its what the CBMC travis jobs did (at least they did the last time I had to work on their travis builds).